airbyte-cdk/java/airbyte-cdk/typing-deduping/src/main/java/io/airbyte/integrations/base/destination/typing_deduping/LoggingUtil.kt

```diff
@@ -0,0 +1,11 @@
+package io.airbyte.integrations.base.destination.typing_deduping
```

Comment:
Kotlin doesn't want you to use underscores in package names [Naming Rules](https://kotlinlang.org/docs/coding-conventions.html#naming-rules):
> Names of packages are always lowercase and do not use underscores (org.example.project). Using multi-word names is generally discouraged, but if you do need to use multiple words, you can either just concatenate them together or use the camel case (org.example.myProject).

While Kotlin itself will still compile with underscores, most kotlin linters/formatters will not you do this.

Reply:
TIL - I am gonna punt on this one though for this version, created a follow up ticket https://github.com/airbytehq/airbyte/issues/34785